.github/workflows/publish-unstable.yml

```diff
@@ -16,17 +16,12 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - name: Install Node
-        uses: actions/setup-node@v3
+      - uses: wyvox/action-setup-pnpm@v2
```

Comment:
note for next step - replace this with `put-built-npm-package-contents-on-branch` per https://github.com/embroider-build/addon-blueprint/blob/main/files/.github/workflows/push-dist.yml

Reply:
I've swapped the `wyvox/action-setup-pnpm@v2` for the `pnpm/action-setup@v3`, as per the blueprint, but I'm not sure how to incorporate the `put-built-npm-package-contents-on-branch` in it. Should it just be added as the next step after `Install dependecies`?

Reply:
@leoeuclids we basically should update this whole file to match https://github.com/embroider-build/addon-blueprint/blob/main/files/.github/workflows/push-dist.yml (replacing placeholders and if/else branching of course) we can do that in follow up PR